antora-playbook-snippets/antora-playbook.yml

```diff
@@ -42,26 +42,21 @@ content:
       branches:
         - main
         - release-1.12.x
-        - release-1.11.x
-        - release-1.10.x
+        - release-2.0.x
       start_path: docs
 
     - url: https://github.com/apache/camel-k-runtime.git
       branches:
         - main
         - release-1.17.x
-        # Old, but required by Camel K 1.11 and 1.10 (to be removed when cleaning)
-        - release-1.16.x
-        - release-1.15.x
+        - release-2.16.x
       start_path: docs
 
     - url: https://github.com/apache/camel-kamelets.git
       branches:
         - main
         - 3.21.x
         - 3.20.x
-        - 0.10.x
```

Comment:
@oscerd are they still maintained? They seem to require older Camel K versions as they had some cross references.

Reply:
I'd say we could remove them, because they were mostly related to camel k and camel-kafka-connector, but for older version.